### ISSUE TITLE
Push date not correct

### ISSUE THREAD

**Issue by Calpicow** [SUSE/Portus#696]
My repos either end up having incorrect push dates or exactly the same as other ones.

**Comment by mssola** [comment]
Does this happen when updating an image on the registry? If so, then it's a duplicate of #553. Otherwise, could you tell me some steps in order to reproduce this? Thanks.

**Comment by Calpicow** [comment]
I don't think it's a duplicate. The problem is that (now that I checked again) all of my images have the exact same push date. The date seem to correspond the most recent activity.

**Comment by lonewulf** [comment]
Hi,
Did you configure the notifications in the registry correctly ?
If not, you're probably seeing portus reconciliating his database with the registry content. All the images would then have the same push date and portus as the author.

**Comment by Calpicow** [comment]
Notifications appears to be working:
```Started POST "/v2/webhooks/events" for 127.0.0.1 at 2016-02-07 02:29:56 +0000
Processing by Api::V2::EventsController#create as JSON
Completed 202 Accepted in 0ms (ActiveRecord: 0.0ms)```

But as you said, the images with the same dates have portus as the author.

**Comment by lonewulf** [comment]
If the images were there before you started using portus, then they were listed by portus and the timestamp is when portus has discovered them. If you pushed an image with the same tag, and the info doesn't update then you're facing #553

**Comment by moix** [comment]
Hi, I'm experiencing the same weird behaviour: a bunch of tags that were uploaded during several days now have same 'Pushed at' time.

On the other hand, the author has changed to 'portus' instead the original one that pushed them.

**Comment by lonewulf** [comment]
Which version are you using ?

There was a bug fixed by #672 and if iirc it was also cherrypicked in the v2.0 branch that looks like your issue.

**Comment by Calpicow** [comment]
I'm using 2.0, but I see that it's fixed in 2.0.1. This may take some time to verify.

**Comment by mssola** [comment]
@Calpicow I cannot reproduce it. If you cannot do it either, I'll simply close this issue. That being said, feel free to re-open this if you can reproduce it :wink:
